mssqlcli/jsonrpc/contracts/connectionservice.py

```diff
@@ -4,6 +4,9 @@
 
```

Comment:
Let's undo this changes to this file since we're only localizing the 'goodbye!' message.